src/graphql/main/schema.graphql

```diff
@@ -283,6 +283,20 @@ type Currency {
   symbol: String!
 }
 
+type DepositFeesInformation {
+  minBankFee: String!
+
+  """
+  below this amount minBankFee will be charged
+  """
+  minBankFeeThreshold: String!
+
+  """
+  ratio to charge as basis points above minBankFeeThreshold amount
+  """
+  ratio: String!
```

Comment:
why type `String` shouldn't this be a custom scalar?

Reply:
because is a bigint :thinking: was setup with that type to do the calcs with payment amounts but let me know if I need to change it to int